src/software/backend/grsim_backend.cpp

```diff
@@ -6,15 +6,16 @@
 
 const std::string GrSimBackend::name = "grsim";
 
-GrSimBackend::GrSimBackend()
-    : network_input(SSL_VISION_DEFAULT_MULTICAST_ADDRESS, SSL_VISION_MULTICAST_PORT,
-                    SSL_GAMECONTROLLER_MULTICAST_ADDRESS,
-                    SSL_GAMECONTROLLER_MULTICAST_PORT,
+GrSimBackend::GrSimBackend(std::shared_ptr<const NetworkConfig> network_config)
+    : network_config(network_config),
+      network_input(network_config->VisionIP()->value(),
```

Comment:
I think we should rename this to `VisionIPv4Address` to be explicit.

Reply:
done